.github/workflows/publish.yml

```diff
@@ -11,6 +11,8 @@ on:  # yamllint disable-line rule:truthy
       - '.github/**'
     tags:
       - "[0-9]+\\.[0-9]+\\.[0-9]+"
```

Comment:
You added the two backslashes here in the previous fix. Sure they are working?

Reply:
The backslashes were used to escape the `.` in the regex, but since GitHub Actions (GHA) uses glob patterns, including them isn't necessary. I've tested it, and it works correctly without the backslashes.

<img width="1296" alt="image" src="https://github.com/user-attachments/assets/f8420fc9-34bd-448e-a83b-e8c6058d9188">